PersonDataRequest/PersonDataRequest/Gallery.swift

```diff
@@ -21,9 +25,9 @@ class Gallery {
     }
     
     func setup() {
-        let img1 = Picture(image: UIImage(named: "img1")!)
-        let img2 = Picture(image: UIImage(named: "img2")!)
-        let img3 = Picture(image: UIImage(named: "img3")!)
+        let img1 = Picture(image: UIImage())
+        let img2 = Picture(image: UIImage())
+        let img3 = Picture(image: UIImage())
```

Comment:
Why first 3 images were deleted and others are still in place?

Reply:
Fixed